### PR TITLE
feat: added read from .env into lido-steth

### PR DESCRIPTION
**Pull request by sergeyWh1te**

### CODE REVIEW

File: ethereum-steth/src/utils/env/env.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    this.grpcPort = parseInt(process.env.AGENT_GRPC_PORT!, 10) || 50051
+    this.httpPort = parseInt(process.env.HTTP_PORT!, 10) || 3000
+    this.logFormat = process.env.LOG_FORMAT || 'simple'
+    this.logLevel = process.env.LOG_LEVEL || 'info'
+
+    this.chainId = parseInt(process.env.FORTA_CHAIN_ID!, 10) || 1
+    this.ethereumRpcUrl = process.env.ETHEREUM_RPC_URL! || 'https://eth.drpc.org'
+
+    this.promPrefix = this.appName.replace('-', '_')
+    this.etherscanKey = Buffer.from('SVZCSjZUSVBXWUpZSllXSVM0SVJBSlcyNjRITkFUUjZHVQ==', 'base64').toString('utf-8')
```
Comment (TheDZhon) [inline ethereum-steth/src/utils/env/env.ts:34]: let's have an env var here too

File: ethereum-steth/.env.sample [hunk 2]
```diff
@@ -0,0 +1,12 @@
+APP_NAME=ethereum-steth
+INSTANCE=forta
+
+HTTP_PORT=3000
+LOG_FORMAT=simple
+LOG_LEVEL=debug
+ETHEREUM_RPC_URL=https://eth.drpc.com
+
```
Comment (TheDZhon) [inline ethereum-steth/.env.sample:8]: let's add `ETHERSCAN_TOKEN`

**Review by TheDZhon — APPROVED** [review]
LGTM :+1:

**Review by TheDZhon — APPROVED** [review]

File: ethereum-steth/src/utils/env/env.ts (hunk 1 above)
Comment (sergeyWh1te) [reply ethereum-steth/src/utils/env/env.ts]: done

**Pull request merged**